<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QbaseClass</class>
 <widget class="QMainWindow" name="QbaseClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qbase</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Connect_btn">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Portname_combo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="Baudrate_combo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>57600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>38400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="Inspector_btn">
    <property name="geometry">
     <rect>
      <x>634</x>
      <y>10</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Inspector</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Cmd_btn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Mode</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Flight_mode_combo">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>STABILIZE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>GUIDED_NOGPS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>LOITER</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>GUIDED</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AUTO</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATL_HOLD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RTL</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>LAND</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CIRCLE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SPORT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DRIFT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>POSHOLD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FLIP</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="Test_btn">
    <property name="geometry">
     <rect>
      <x>634</x>
      <y>40</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Test</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Arm_btn">
    <property name="geometry">
     <rect>
      <x>337</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Arm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setMarker_btn">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>170</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>显示定位</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LngEdit">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>70</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LatEdit">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>120</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>758</x>
      <y>70</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>经度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>758</x>
      <y>120</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>纬度：</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>701</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="clearMarker_btn">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>210</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>清除标记</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="HightEdit">
    <property name="geometry">
     <rect>
      <x>428</x>
      <y>10</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Takeoff_btn">
    <property name="geometry">
     <rect>
      <x>513</x>
      <y>8</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Take Off</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>337</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>起飞高度(m)：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FlyToHere_btn">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>170</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>飞到此处</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Choose_id_combo">
    <property name="geometry">
     <rect>
      <x>748</x>
      <y>250</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>220</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>选择飞机编号</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FlyWest_btn">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>430</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>往西飞</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FlyNorth_btn">
    <property name="geometry">
     <rect>
      <x>801</x>
      <y>400</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>往北飞</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FlyEast_btn">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>430</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>往东飞</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FlySouth_btn">
    <property name="geometry">
     <rect>
      <x>802</x>
      <y>460</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>往南飞</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="VelEdit">
    <property name="geometry">
     <rect>
      <x>801</x>
      <y>364</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>766</x>
      <y>366</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>速度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="Info_label">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>490</y>
      <width>161</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>861</x>
      <y>366</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>m/s</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Land_btn">
    <property name="geometry">
     <rect>
      <x>513</x>
      <y>40</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Land</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Disarm_btn">
    <property name="geometry">
     <rect>
      <x>426</x>
      <y>39</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Disarm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FlyUp_btn">
    <property name="geometry">
     <rect>
      <x>753</x>
      <y>320</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>往上飞</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FlyDown_btn">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>320</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>往下飞</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>925</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionMavlink_Inspector">
   <property name="text">
    <string>Mavlink Inspector</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Qbase.qrc"/>
 </resources>
 <connections/>
</ui>
